### PR TITLE
Input

### PR DESCRIPTION
**Pull request by MicaQuarleri**
Shared input was created and added to every entity.

### CODE REVIEW

**Review by leimeterjoaquin — APPROVED** [review]
Great!

File: src/Components/Councelors/Form/FormCounselor.js [hunk 1]
```diff
@@ -36,6 +36,50 @@ const Form = () => {
...
+  const params = new URLSearchParams(window.location.search);
+  const councelorId = params.get('id');
+
+  if (councelorId) {
+    useEffect(() => {
+      fetch(`${process.env.REACT_APP_API}/counselors/id/${councelorId}`)
+        .then((response) => response.json())
+        .then((response) => {
+          onLoading(response);
+          console.log(response);
```
Comment (aquilesmariani) [inline src/Components/Councelors/Form/FormCounselor.js:48]: Please remove this console.log

File: src/Components/Councelors/Form/FormCounselor.js [hunk 2]
```diff
@@ -229,12 +271,8 @@ const Form = () => {
...
   const hidePhone = () => {
     setPhoneError(false);
-    if (
-      phoneValue.length < 8 ||
-      phoneValue.includes('(') ||
-      phoneValue.includes(')') ||
-      phoneValue.includes('#')
-    ) {
+    if (phoneValue.length < 8) {
+      console.log(phoneValue);
```
Comment (aquilesmariani) [inline src/Components/Councelors/Form/FormCounselor.js:275]: Remove this one as well

**Review by aquilesmariani — CHANGES_REQUESTED** [review]
I think we should rename the form component filenames so they are just `index.js` since we are already providing the entity name and the Form type of component in the path (e.g `/Components/Interviews/Form`)
Please refer to the message I left on slack for more info.

**Review by aquilesmariani — APPROVED** [review]
Looks nice!

**Pull request merged**